packages/@okta/vuepress-site/guides/sign-into-spa/section09/react/reqauthspecific.md

```diff
@@ -1 +1,21 @@
-placeholder
\ No newline at end of file
+We provide a `SecureRoute` component which extends the `Route` component to require authentication. Use `SecureRoute` on any routes which require authentication.
```

Comment:
We provide a `SecureRoute` component that extends the `Route` component to require authentication. Use `SecureRoute` on any routes that require authentication.